src/sentry/static/sentry/app/icons/index.jsx

```diff
@@ -0,0 +1,73 @@
+export {IconActivity} from './iconActivity';
```

Comment:
Actually this should be `tsx` otherwise we won't have types on these.

Reply:
Got it